src/plugins/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -298,10 +298,12 @@ export class SearchInterceptor {
           isSavedToBackground = true;
         });
 
-    const cancel = once(() => {
-      if (id && !isSavedToBackground) this.deps.http.delete(`/internal/search/${strategy}/${id}`);
+    const sendCancelRequest = once(() => {
+      this.deps.http.delete(`/internal/search/${strategy}/${id}`, { version: '1' });
     });
 
+    const cancel = () => id && !isSavedToBackground && sendCancelRequest();
```

Comment:
This changes a bit the previous behaviour, but it looks more correct to me to fire off the cancel request only if all checks pass. Have we ever had issues with the previous check?

Reply:
No, we haven't had any issues that I'm aware of, but I noticed the same thing when going through this code.